ProjectSettings/TimelineSettings.asset

```diff
@@ -0,0 +1,16 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &1
+MonoBehaviour:
```

Comment:
@ReubenBaker Heya do you have any idea about this file? did you change any settings anywhere?